File: Linear_search/R/linear_search.r
arr <- c(123, 62, 52111, 9891, 1)
target <- 1
index <- which(arr == target)

if (length(index) > 0) {
    i <- index[1]
    cat("The element", target, "was found at position", i, "\n")
} else {
    cat("The element", target, "was not found\n")
}

File: Linear_search/Lua/linear_search.lua
local arr = {151, 121, 5, 111114, 23}
local target = 111114
local index = nil

for i = 1, # arr do
  if arr[i] == target then
    index = i
    break
  end
end

if index then 
  print("The element " .. target .. " was found at position " .. index)
else 
  print("The element " .. target .. " was not found")  
end  
